hal/lib/athena/HAL.cpp

```diff
@@ -7,11 +7,11 @@
 
 #include "HAL/HAL.h"
 
-#include <signal.h>  // linux for kill
-#include <stdlib.h>
 #include <sys/prctl.h>
 #include <unistd.h>
 
+#include <csignal>  // linux for kill
```

Comment:
http://en.cppreference.com/w/cpp/header/csignal doesn't have kill(), so I forgot to add it.